src/Workspaces/Core/Portable/TodoComments/TodoCommentOptions.cs

```diff
@@ -2,27 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Immutable;
-using System.Composition;
-using Microsoft.CodeAnalysis.Host.Mef;
+using System.Runtime.Serialization;
 using Microsoft.CodeAnalysis.Options;
-using Microsoft.CodeAnalysis.Options.Providers;
 
-namespace Microsoft.CodeAnalysis.Editor.Implementation.TodoComments
+namespace Microsoft.CodeAnalysis.TodoComments
 {
-    [ExportSolutionOptionProvider, Shared]
-    internal sealed class TodoCommentOptions : IOptionProvider
+    [DataContract]
+    internal readonly record struct TodoCommentOptions(
+        [property: DataMember(Order = 0)] string TokenList = "")
```

Comment:
why the optional value?

Reply:
General pattern, where all parameters are optional and their defaults are the options' default values.